va/va.h

```diff
@@ -3437,6 +3437,28 @@ VAStatus vaMapBuffer (
     void **pbuf 	/* out */
 );
 
+/**
+ * Map data store of the buffer into the client's address space
+ * this interface could be used to convey the operation hint
+ * backend driver could use these hint to optimize the implementations
+ */
+typedef enum
+{
+    /** \brief VAMapBufferFlagDefault is used when there are no flag specified*/
+    VAMapBufferFlagDefault = 0,
+    /** \brief application will read the surface after map */
```

Comment:
Can the flags be used on OR ? It's not clear from text description.

Reply:
+1. Not clear whether this `VAMapBufferFlagRead | VAMapBufferFlagWrite` can be done. For example:
```
/** \brief Bitwise flag: application will read the surface after map */
```

and same below for write:
```
/** \brief Bitwise flag: application will write the surface after map */
```